docs/reference/configurations.md

```diff
@@ -0,0 +1,81 @@
+# Configurations
+
+- _blog_hostname_: Hostname for accessing WordPress, if ingress relation is active. Defaults to the
+  application name.
+- _db_host_: MySQL database host. Takes precedence over database relation if set. Must be set with
+  a complete set of database configurations to take effect.
+- _db_name_: MySQL database name. Takes precedence over database relation if set. Must be set with
+  a complete set of database configurations to take effect.
+- _db_user_: MySQL database user. Takes precedence over database relation if set. Must be set with
+  a complete set of database configurations to take effect.
+- _db_password_: MySQL database user’s password. Takes precedence over database relation if set.
+  Must be set with a complete set of database configurations to take effect.
+- _initial_settings_: YAML formatted WordPress configuration. It is used only
+  during initial deployment. Changing it at a later stage has no effect.
+  If set to non empty string required keys are:
+
+  ```yaml
+  user_name: admin_username
+  admin_email: name@example.com
+  ```
+
+  Optionally you can also provide
+
+  ```yaml
+  admin_password: <secret> # auto generated if not set
+  blog_public: False # by default blogs are public
+  ```
+
+  If _admin_password_ is not provided it will be automatically generated and stored on the operator
+  pod in the /root directory.
+
+- _plugins_: Plugin slugs of plugins to be installed, separated by comma. Including or excluding a
+  default plugin here has no effect.
+- _themes_: Theme slugs of themes to be installed, separated by comma. Including or excluding a
+  default theme here has no effect.
+- _wp_plugin_akismet_key_: Akismet anti-spam plugin key. If empty, akismet will not be enabled
+  automatically.
+- _wp_plugin_openid_team_map_: Launchpad teams and corresponding access levels, for use with the
+  openid plugins.
+
+  Valid WordPress access levels are: administrator, editor, author, contributor, subscriber
+
+  If empty, OpenID will not be enabled.
+
+  Format is key=value pairs (where key is the Launchpad team, and value is
+  the WordPress role) - commas separate multiple pairs.
+
+  Example format:
+
+  "site-sysadmins=administrator,site-editors=editor,site-executives=editor"
+
+- _wp_plugin_openstack_objectstorage_config_: YAML dictionary with keys named after WordPress
+  settings and the desired values. Please note that the settings will be reset to values provided
+  every time hooks run. Below are available values of the configuration yaml.
+
```

Comment:
Maybe we should mention that the openstack object storage plugin is a must for multi-unit deployment?

Reply:
Sounds good, added!